packages/grafana-ui/src/components/Select/getSelectStyles.ts

```diff
@@ -22,6 +22,7 @@ export const getSelectStyles = stylesFactory((theme: GrafanaTheme) => {
       display: flex;
       align-items: center;
       flex-direction: row;
+      flex-shrink: 0;
```

Comment:
What does this do? does select still support smaller width than the value? (for example, select a data source with really long name in data source picker)

Reply:
It prevents the Select options from being squished vertically. Another option is to remove `display: flex` and default to `block`, but I guess it's there for a reason.

Tested with a few Selects and didn't notice any issues. 


Before:
![Screenshot 2020-05-12 at 11 22 11](https://user-images.githubusercontent.com/8878045/81660616-a6c6fd80-9443-11ea-84f4-44e9cde84167.png)
After:
![Screenshot 2020-05-12 at 11 25 38](https://user-images.githubusercontent.com/8878045/81660624-ac244800-9443-11ea-9266-d9067aa0b99d.png)